src/main/scala/nl/biopet/tools/sampletsvtoconfig/SamplesTsvToConfig.scala

```diff
@@ -52,9 +52,10 @@ object SamplesTsvToConfig extends ToolCommand {
           if (libraryColumn != -1) Some(values(libraryColumn)) else None
 
         //FIXME: this is a workaround, should be removed after fixing #180
-        if (sample.head.isDigit || library.exists(_.head.isDigit))
-          throw new IllegalStateException(
-            "Sample or library may not start with a number")
+        //if (sample.head.isDigit || library.exists(_.head.isDigit))
+        //  throw new IllegalStateException(
+        //    "Sample or library may not start with a number")
+        // Fixed ?
```

Comment:
Not sure ;)

Reply:
@ffinfo I have updated the branch to reflect the comments, but the commits do not show up here.